### ISSUE TITLE
Running Holos with no command throws an error

### ISSUE THREAD

**Issue by MikeIvanichev** [holos-run/holos#334]
Running holos with no command throws an error.

 ```
❯ holos
panic: runtime error: slice bounds out of range [2:1]

goroutine 1 [running]:
github.com/holos-run/holos/internal/cli.newCueCmd(...)
        /home/mike/go/pkg/mod/github.com/holos-run/holos@v0.98.1/internal/cli/root.go:121
github.com/holos-run/holos/internal/cli.New(0xc0002837c0, {0x3826e00, 0x4f60860})
        /home/mike/go/pkg/mod/github.com/holos-run/holos@v0.98.1/internal/cli/root.go:102 +0x772
main.main.MakeMain.func1()
        /home/mike/go/pkg/mod/github.com/holos-run/holos@v0.98.1/internal/cli/main.go:22 +0x5b
main.main()
        /home/mike/go/pkg/mod/github.com/holos-run/holos@v0.98.1/cmd/holos/main.go:10 +0x3e
```

```
❯ holos -v
0.98.1
```

**Comment by jeffmccune** [comment]
Thanks for reporting this, taking a look.

**Comment by jeffmccune** [comment]
Fixed in 0.98.2
